src/components/MainWelcome.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+
+import { UserDisplay } from '../constants/Types';
+import './MainWelcome.css';
+
+type MainWelcomeProps = {
+    userDisplay: UserDisplay;
+};
+
+const MainWelcome: FC<MainWelcomeProps> = (props: MainWelcomeProps) => {
+    return (
+        <div className="main-welcome-container">
+            <div>
```

Comment:
I think we should have a separate component for "header" and "primary-text" and "secondary-text", so that the component can style the letters and have a consistent font, font size, and weight for each time we use some kind of text.

Reply:
Do you think it'd be better to have a separate component or just a shared css class?